samples/projects/cdktf-typescript/package.json

```diff
@@ -21,14 +21,12 @@
     "test:watch": "jest --watch",
     "upgrade": "npm i cdktf@latest cdktf-cli@latest",
     "upgrade:next": "npm i cdktf@next cdktf-cli@next",
-    "ci-build": "npm ci && npm test && npm run synth"
+    "ci-build": "npm ci && npm test && npm run synth",
+    "update-local-cdfpackage": "rm -rf node_modules/@openfabr/package-ri-cdktf && npm i @openfabr/package-ri-cdktf ../../packages/cdktf-typescript -f"
```

Comment:
What does it do in the context of this package?

Reply:
Dev time thing. During package dev interation, when directly referencing the package for testing in this project for local dev, cleanly pulls in the latest version.